packages/react/radio-group/src/RadioGroup.tsx

```diff
@@ -87,14 +89,13 @@ const RadioGroup = React.forwardRef<RadioGroupElement, RadioGroupProps>(
           asChild
           {...rovingFocusGroupScope}
           orientation={orientation}
-          dir={dir}
+          dir={direction}
           loop={loop}
         >
           <Primitive.div
             role="radiogroup"
             aria-orientation={orientation}
             aria-labelledby={labelledBy}
-            dir={dir}
```

Comment:
This isn't needed because `RovingFocusGroup` adds it. We haven't added it in other places for that reason but not sure what is the best approach. I've gone with removing it throughout this PR for now but happy to put back in if we feel that's best.